weaviate/collections/base.py

```diff
@@ -12,17 +13,59 @@
     _collection_configs_from_json,
     _collection_configs_simple_from_json,
 )
-from weaviate.exceptions import UnexpectedStatusCodeException
+from weaviate.exceptions import EmptyResponseException, UnexpectedStatusCodeException
 from weaviate.util import _capitalize_first_letter, _decode_json_response_dict
 
 if TYPE_CHECKING:
     from weaviate.collections.batch.executor import BatchExecutor
 
 
 class _CollectionBase:
-    def __init__(self, name: str) -> None:
+    def __init__(self, connection: Connection, name: str) -> None:
+        self._connection = connection
         self.name = _capitalize_first_letter(name)
 
+    def shards(
+        self,
+    ) -> List[Shard]:
+        """
+        Get the statuses of all the shards of this collection.
+
+        If the collection is multi-tenancy, each shard is one of the tenants. If the collection is single-tenancy, there is only one shard.
+
+        Arguments:
+            `output`
+                Set the desired output verbosity level. Can be `[minimal | verbose]`, defaults to server-side default of `minimal`.
+
+        Returns:
+            The list of shards belonging to this collection.
+
+        Raises
+            `requests.ConnectionError`
+                If the network connection to weaviate fails.
+            `weaviate.UnexpectedStatusCodeException`
+                If weaviate reports a none OK status.
+            `weaviate.EmptyResponseException`
+                If the response is empty.
+        """
+        path = f"/nodes/{_capitalize_first_letter(self.name)}?output=verbose"
```

Comment:
you just hardcode the value here

Reply:
This is because the shards field will be `null` if `output=minimal`. Since this `shards()` method is used to inspect all the shards belonging to this collection, we want the verbose response in order to not receive `null`

Reply:
just added it because the parameter was in the docstring - so I was not sure if the docstring was wrong or the missing paramter was wrong